### PR TITLE
3914 Only Show Edit Once QCed

### PR DESCRIPTION
**Pull request by wbyoko**

### CODE REVIEW

File: web-client/integration-tests/docketClerkEditsDocketEntryMeta.test.js [hunk 1]
```diff
@@ -26,20 +26,16 @@ describe("Docket Clerk Edits a Docket Entry's Meta", () => {
   petitionerChoosesProcedureType(test, { procedureType: 'Regular' });
   petitionerChoosesCaseType(test);
   petitionerCreatesNewCase(test, fakeFile);
+  petitionerFilesADocumentForCase(test, fakeFile);
   petitionerSignsOut(test);
 
   docketClerkLogIn(test);
-  docketClerkCreatesAnOrder(test, {
```
Comment (matthopson) [inline web-client/integration-tests/docketClerkEditsDocketEntryMeta.test.js:32]: Is there already a draft order created without having to explicitly create one?

Comment (wbyoko) [reply web-client/integration-tests/docketClerkEditsDocketEntryMeta.test.js:32]: Does there need to be a draft order?

Comment (matthopson) [reply web-client/integration-tests/docketClerkEditsDocketEntryMeta.test.js:32]: Sorry, one that hasn't been QC'd yet.

Comment (wbyoko) [reply web-client/integration-tests/docketClerkEditsDocketEntryMeta.test.js:32]: Or just a docket entry created by adding a document?

Comment (wbyoko) [reply web-client/integration-tests/docketClerkEditsDocketEntryMeta.test.js:32]: petitionerFilesADocumentForCase was added for this purpose

**Review by matthopson — APPROVED** [review]

**Pull request merged**